integration-tests/maven/src/test/java/io/quarkus/maven/CreateExtensionMojoTest.java

```diff
@@ -45,6 +45,22 @@ static CreateExtensionMojo createMojo(String testProjectName) throws IllegalArgu
         return mojo;
     }
 
+    private static Path getCopyDir(String testProjectName) {
+        Random random = new Random();
```

Comment:
I think this would be better as static, don't you think?

Reply:
I've always been a little hazy about whether Random should be `static` or not. Let me read up

Reply:
It looks like having it as an argument will cause it to use a new seed which should theoretically make it more random. 
In any case it's a super small issue, I don't mind doing it either way

Reply:
Shouldn't we use `UUID.randomUUID()` instead?

Reply:
Since this is the second time this has come up, I'll go ahead and do it :P

Reply:
@geoand you don't want it to be super random because you don't want conflicts :). Using only one `Random` is recommended.

@mkouba why not? @machi1990 was suggesting this too.

Reply:
`UUID.randomUUID()` it is :)